soh/soh/Enhancements/randomizer/ShuffleTrees.cpp

```diff
@@ -0,0 +1,212 @@
+    locationTable[RC_HF_SOUTHEAST_TREE_6]         = Location::Tree(RC_HF_SOUTHEAST_TREE_6,               RCQUEST_BOTH, RCAREA_HYRULE_FIELD,          SCENE_HYRULE_FIELD,                TWO_ACTOR_PARAMS(549, 11204),   "Tree in Southeast 6",               RHT_TREE_HYRULE_FIELD,    RG_BLUE_RUPEE,  SpoilerCollectionCheck::RandomizerInf(RAND_INF_HF_SOUTHEAST_TREE_6));
+    locationTable[RC_HF_SOUTHEAST_TREE_7]         = Location::Tree(RC_HF_SOUTHEAST_TREE_7,               RCQUEST_BOTH, RCAREA_HYRULE_FIELD,          SCENE_HYRULE_FIELD,                TWO_ACTOR_PARAMS(49, 11405),    "Tree in Southeast 7",               RHT_TREE_HYRULE_FIELD,    RG_BLUE_RUPEE,  SpoilerCollectionCheck::RandomizerInf(RAND_INF_HF_SOUTHEAST_TREE_7));
+    locationTable[RC_HF_SOUTHEAST_TREE_8]         = Location::Tree(RC_HF_SOUTHEAST_TREE_8,               RCQUEST_BOTH, RCAREA_HYRULE_FIELD,          SCENE_HYRULE_FIELD,                TWO_ACTOR_PARAMS(-29, 12005),   "Tree in Southeast 8",               RHT_TREE_HYRULE_FIELD,    RG_BLUE_RUPEE,  SpoilerCollectionCheck::RandomizerInf(RAND_INF_HF_SOUTHEAST_TREE_8));
+    locationTable[RC_HF_SOUTHEAST_TREE_9]         = Location::Tree(RC_HF_SOUTHEAST_TREE_9,               RCQUEST_BOTH, RCAREA_HYRULE_FIELD,          SCENE_HYRULE_FIELD,                TWO_ACTOR_PARAMS(1514,13157),   "Tree in Southeast 9",               RHT_TREE_HYRULE_FIELD,    RG_BLUE_RUPEE,  SpoilerCollectionCheck::RandomizerInf(RAND_INF_HF_SOUTHEAST_TREE_9));
+    locationTable[RC_HF_SOUTHEAST_TREE_10]        = Location::Tree(RC_HF_SOUTHEAST_TREE_10,              RCQUEST_BOTH, RCAREA_HYRULE_FIELD,          SCENE_HYRULE_FIELD,                TWO_ACTOR_PARAMS(-509,12954),   "Tree in Southeast 10",              RHT_TREE_HYRULE_FIELD,    RG_BLUE_RUPEE,  SpoilerCollectionCheck::RandomizerInf(RAND_INF_HF_SOUTHEAST_TREE_10));
+    locationTable[RC_HF_SOUTHEAST_TREE_11]        = Location::Tree(RC_HF_SOUTHEAST_TREE_11,              RCQUEST_BOTH, RCAREA_HYRULE_FIELD,          SCENE_HYRULE_FIELD,                TWO_ACTOR_PARAMS(-430,12354),   "Tree in Southeast 11",              RHT_TREE_HYRULE_FIELD,    RG_BLUE_RUPEE,  SpoilerCollectionCheck::RandomizerInf(RAND_INF_HF_SOUTHEAST_TREE_11));
+    locationTable[RC_HF_SOUTHEAST_TREE_12]        = Location::Tree(RC_HF_SOUTHEAST_TREE_12,              RCQUEST_BOTH, RCAREA_HYRULE_FIELD,          SCENE_HYRULE_FIELD,                TWO_ACTOR_PARAMS(69,12153),     "Tree in Southeast 12",              RHT_TREE_HYRULE_FIELD,    RG_BLUE_RUPEE,  SpoilerCollectionCheck::RandomizerInf(RAND_INF_HF_SOUTHEAST_TREE_12));
+    locationTable[RC_HF_SOUTHEAST_TREE_13]        = Location::Tree(RC_HF_SOUTHEAST_TREE_13,              RCQUEST_BOTH, RCAREA_HYRULE_FIELD,          SCENE_HYRULE_FIELD,                TWO_ACTOR_PARAMS(-129,12554),   "Tree in Southeast 13",              RHT_TREE_HYRULE_FIELD,    RG_BOMBS_5,     SpoilerCollectionCheck::RandomizerInf(RAND_INF_HF_SOUTHEAST_TREE_13));
```

Comment:
instead of doing 3x3 scan can put a special `if` for this & investigate more why this tree is finicky